src/main/java/aluczak/reservationsystem/ReservationController.java

```diff
@@ -0,0 +1,178 @@
+package aluczak.reservationsystem;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.BindingResult;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+import javax.validation.Valid;
+import java.math.BigDecimal;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import java.util.Optional;
+
+@RestController
+public class ReservationController {
+    @Autowired
+    MovieRestRepository movieRestRepository;
+    @Autowired
+    ReservationRepository reservationRepository;
+    @Autowired
+    TicketRepository ticketRepository;
+    @Autowired
+    SeatRepository seatRepository;
+
+
+    @PostMapping(value = "/makeReservation",
+            headers = "Accept=application/json;charset=utf-8",
```

Comment:
Hmmm, header Accept w zwrotce?

Reply:
afaik to nie jest zwrotka... tak czy siak to się chyba robi za pomocą `produces`